static/app/views/starfish/utils/getDateFilters.tsx

```diff
@@ -1,9 +1,13 @@
-import moment from 'moment';
+import moment, {Moment} from 'moment';
 
 export const PERIOD_REGEX = /^(\d+)([h,d])$/;
 export const DATE_FORMAT = 'YYYY-MM-DD HH:mm:ss';
 
-export function getDateFilters(pageFilter) {
+export function getDateFilters(pageFilter): {
+  endTime: Moment;
+  startTime: Moment;
+  statsPeriod: string;
+} {
```

Comment:
Were the types inferred incorrectly before this?

Reply:
previously statsPeriod was inferred as `any` because the pageFilter parameter is of type `any`.
 Probs would be better to just type pageFilter, but I wasn't sure how to type it properly because it comes from the `PageFiltersStore`.

Reply:
https://github.com/getsentry/sentry/pull/51157